giphy-search/src/components/GifContainer.jsx

```diff
@@ -1,4 +1,17 @@
-function GifContainer() {
+/* 
+This component is meant to display gifs. However, there are two different sets of gifs that this component can display, depending on the user's actions. At first, they should see trending gifs but after submitting a search term in the GifSearch form, they should see gifs related to their search.
+
+TODO:
+- use the getTrendingGifs adapter to fetch trending gifs on the first render
+- each time the user submits the form in GifSearch, use the getGifsBySearch adapter to fetch gifs according to the search term.
+- render the list of fetched gifs (or the defaultGifs) as list items with an `img` inside. Remember to give each list item a unique key!
+- Bonus: if at any point an error is returned, render the default gifs again.
+*/
+
+import defaultGifs from '/gifs.json';
```

Comment:
Adding in the defaultGifs here as an option for fellows who are struggling to get fetching to work. At least they will have data that they can practice rendering a list with.

They can use this as the default `useState` value for their `gifs` as well prior to fetching